tests/unit_tests/test_wave_03.py

```diff
@@ -130,8 +130,8 @@ def test_swap_items_from_their_empty_returns_false():
     nobodys_item = Item()
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
-
-    raise Exception("Complete this test according to comments below.")
+    assert result == False
```

Comment:
If a swap didn't occur, we need to check that the items that were in the vendor's inventory are still there and that nothing was accidentally added to the other vendor's inventory.

What other assertions would you write? Think about it and then have a look at what some possibilities are below.

```py
    assert nobodys_item not in fatimah.inventory
    assert item_a in fatimah.inventory
    assert item_b in fatimah.inventory
    assert item_c in fatimah.inventory
    assert len(jolie.inventory) == 0
    assert len(fatimah.inventory) == 3